cumulus_library/template_sql/base_templates.py

```diff
@@ -212,6 +211,8 @@ def get_insert_into_query(
     :param table_cols: Comma deleniated column names, i.e. ['first','second']
     :param dataset: Array of data arrays to insert, i.e. [['1','3'],['2','4']]
     """
+    if type_casts is None:
+        type_casts = {}
```

Comment:
nit: might be a matter of taste, but I think `type_casts = type_casts or {}` is more readable / common. But this works too. (here and elsewhere)